docs/developing.rst

```diff
@@ -5,7 +5,7 @@ edX Proctoring Developer Guide
 
 
 How do I use proctoring on devstack?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+------------------------------------
```

Comment:
By way of explanation, we skipped a section heading level I found I wanted to use